ts/features/wallet/onboarding/privative/screens/add/AddPrivativeCardComponent.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from "react";
+import { SafeAreaView, StyleSheet } from "react-native";
+import { View } from "native-base";
+import { ScrollView } from "react-native-gesture-handler";
+import I18n from "../../../../../../i18n";
+import { PaymentInstrument } from "../../../../../../../definitions/pagopa/walletv2/PaymentInstrument";
+import BaseScreenComponent from "../../../../../../components/screens/BaseScreenComponent";
+import { IOStyles } from "../../../../../../components/core/variables/IOStyles";
+import { H1 } from "../../../../../../components/core/typography/H1";
+import {
+  getPrivativeGdoLogoUrl,
+  getPrivativeLoyaltyLogoUrl,
+  getTitleFromPaymentInstrument,
+  isCoBadgeOrPrivativeBlocked
+} from "../../../../../../utils/paymentMethod";
+import { InfoBox } from "../../../../../../components/box/InfoBox";
+import { IOColors } from "../../../../../../components/core/variables/IOColors";
+import { Body } from "../../../../../../components/core/typography/Body";
+import FooterWithButtons from "../../../../../../components/ui/FooterWithButtons";
+import {
+  cancelButtonProps,
+  confirmButtonProps
+} from "../../../../../bonus/bonusVacanze/components/buttons/ButtonConfigurations";
+import BasePrivativeCard from "../../../../privative/component/card/BasePrivativeCard";
+
+type Props = {
+  pan: PaymentInstrument;
```

Comment:
What you think about  renaming `pan` to `pm` or `paymentInstrument`? Just to avoid clash with the meaning of pan.

Reply:
yes sure, modified in [7d09c40](https://github.com/pagopa/io-app/pull/2916/commits/7d09c405598557b587fa521f3b3d9e08f8b7695f)